src/io/flutter/run/FlutterAppState.java

```diff
@@ -58,8 +61,6 @@ protected ProcessHandler startProcess() throws ExecutionException {
     FlutterDaemonService service = FlutterDaemonService.getInstance();
     assert service != null;
     Project project = getEnvironment().getProject();
-    String workingDir = project.getBasePath();
-    assert workingDir != null;
```

Comment:
Shouldn't we ensure the Project is not the default project before starting the daemon?